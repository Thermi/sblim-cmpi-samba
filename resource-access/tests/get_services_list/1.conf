scriptdir ="../../scripts"
smbconf   ="../../smb.sample"

