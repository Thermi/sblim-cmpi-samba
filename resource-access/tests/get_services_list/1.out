Section: global
Section: myshare
Section: users
Section: print$
Section: printers
